crates/bevy_pbr/src/render/utils.wgsl

```diff
@@ -1,6 +1,6 @@
 #define_import_path bevy_pbr::utils
 
-const PI: f32 = 3.141592653589793;
+const PI: f32 = 3.14159265358979323846264338327950288;
```

Comment:
Nitpick: This seems like excessive precision for `f32`, even the existing constant before the change was already more than what can fit in a single precision float.

Reply:
Reverting this. It appears that the previous float is the maximum precision for IEEE-754.